proposal.rst

```diff
@@ -268,7 +267,29 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+* Weeks of bonding: get familiar with ranking algorithms, search for suitable datasets, assessment of the complexity of integrating large databases, and performance tests.
+
+In each next position assumes that testing and writing cross-platform part is included.
+
+* 1 Week - integrate datasets 
+
+* 2 Weeks of exams (situation with exams is very unstable now in my University, so I timeline may change in this case)
+
+* 1 Week - solve possible problems with the size of datasets
+
+* 3 Weeks - implement ListMLE, ListNet and SVM evaluation api
+
+If no troubles appeared, I can try to develop suite for performance testing:
+
+* 1 Week - get familiar with data indexing and searching api
+
+* 1.5 Weeks - tests for indexing
+
+* 1.5 Weeks - tests for searching
```

Comment:
Again, we expect to see more detail. What are you actually aiming to test here? The project description mentions time taken, memory used, and disk space used — but there are other options (such as user + system CPU time) that could be useful as well.

As with the evaluation side, what dataset(s) do you want to work with?